### PR TITLE
fix: create and join mixnodes families

### PR DESCRIPTION
**Pull request by sven-hash**

### CODE REVIEW

**Comment by sven-hash** [comment]
I think it should be better to explain how to use `.env` file instead of copy paste the mnemonic in the terminal since it will stay in bash history

**Comment by mfahampshire** [comment]
Agree re the `.env`, good idea. 

I have a question re: why you've switched over to use the `nym-cli` instead of `nyxd` - perhaps you could edit the PR and have both options? There will be some people wanting to use `nyxd` imo.

**Comment by sven-hash** [comment]
Because I didn't find the nyxd binary on the latests release i.e https://github.com/nymtech/nym/releases/tag/nym-binaries-v1.1.9 so I deducted that nym-cli replace nyxd

Let me know what you think

**Comment by mfahampshire** [comment]
No we don't do a release of the `nyxd` binary, we expect people to compile it themselves right now. `nym-cli` is a nice wrapper tool but not everyone uses it.

**Comment by sven-hash** [comment]
Added nyxd, but could you please check that the command is correct and maybe add the mixnet contract and validator-variables values

You have the right to edit the file, feel free to adapt

File: src/nodes/mix-node-setup.md [hunk 1]
```diff
@@ -425,30 +425,123 @@ Then reboot your server and restart your mixnode.
...
+
+
+
+      _ __  _   _ _ __ ___
+     | '_ \| | | | '_ \ _ \
+     | | | | |_| | | | | | |
+     |_| |_|\__, |_| |_| |_|
+            |___/
+
+             (nym-mixnode - version 1.1.9)
```
Comment (mfahampshire) [inline src/nodes/mix-node-setup.md]: replace version number with variable

File: src/nodes/mix-node-setup.md [hunk 2]
```diff
@@ -425,30 +425,123 @@ Then reboot your server and restart your mixnode.
...
 
-This will return a signature which is going to be used below - in this example it is `3SEjfNcJ5L3cXdvWCdiQNT5DkCFJ2TurK5xsYyEdHH324nAA3bWvKoXmkjU9Xbr9ZyemGDLJ4dmGEHWUwL1LCWKq`. 
+~~~admonish example collapsible=true title="Console output"
+      _ __  _   _ _ __ ___
+     | '_ \| | | | '_ \ _ \
+     | | | | |_| | | | | | |
+     |_| |_|\__, |_| |_| |_|
+            |___/
+
+             (nym-mixnode - version 1.1.9)
```
Comment (mfahampshire) [inline src/nodes/mix-node-setup.md]: ditto

File: src/nodes/mix-node-setup.md [hunk 3]
```diff
@@ -425,30 +425,123 @@ Then reboot your server and restart your mixnode.
...
+
+    
+Signing the text "4yRfauFzZnejJhG2FACTVQ7UnYEcFUYw3HzXrmuwLMaR" using your mixnode's Ed25519 identity key...
+The base58-encoded signature on '4yRfauFzZnejJhG2FACTVQ7UnYEcFUYw3HzXrmuwLMaR' is: 4By7EQEMM8BAt6ptxJyeGqpoxHWxeRUhyJ4wMr2x3mXSQD9nvttkvd7tgP1uKu2ktJjB2bLzD1oaZ33d2Wv5eYWp
+~~~
+
+The `--mnemonic` is going to be the mnemonic of the member wanting to join the family: 
 
-The `--from` is going to be the mnemonic of the member wanting to join the family:
```
Comment (mfahampshire) [inline src/nodes/mix-node-setup.md:438]: looks like this line was deleted and not moved anywhere? Its important , so make sure its re-included if its disappeared by accident

**Review by mfahampshire — CHANGES_REQUESTED** [review]

**Pull request merged**